components/dataflow/tfma/src/model_analysis.py

```diff
@@ -224,6 +224,11 @@ def generate_static_html_output(output_dir, slicing_columns):
 def main():
   tf.logging.set_verbosity(tf.logging.INFO)
   args = parse_arguments()
+  args.slice_columns = [
+    column
+    for column in column_group.split(',')
+    for column_group in args.slice_columns
```

Comment:
nit: If we support the split by ',', is it still needed to make slice_columns as an append arg? Maybe a single string arg is less confusing.

Reply:
I agree with you, but I did not want to change the command-line API.
Maybe I should change it since I'm not sure anyone is using that multi-column functionality right now.